components/article/client/view.coffee

```diff
@@ -58,11 +58,11 @@ module.exports = class ArticleView extends Backbone.View
     @setupStickyShare()
     @setupFollowButtons()
     @setupImageSets()
+    @setupMarketoStyles() if @article.attributes.channel and @article.attributes.channel.type is "team"
```

Comment:
👍  nice guard statement here. You can also shorten this with Coffeescript's existential operator

`if @article.attributes.channel?.type is "team"`